packages/protocol/lib/signing-utils.ts

```diff
@@ -26,6 +26,16 @@ function makeEven(hex: string) {
   return hex
 }
 
+export const getParsedSignatureOfAddress = async (web3: Web3, address: string, signer: string) => {
```

Comment:
Do we have this in @celo/utils?

Reply:
We do not

Reply:
Should we? Seems to be used many places

Reply:
I was thinking about it, but right now it was nice to not require the web3 module in it, so I just left it here for now